app/controllers/votes_controller.rb

```diff
@@ -3,22 +3,15 @@ class VotesController < ApplicationController
 
   def create
     vote = current_quote.votes.new vote_params.merge(user: current_user)
-    if vote.save
-      flash[:notice] = "Voted!"
-    else
-      flash[:alert] = "Something went wrong!"
-    end
-    redirect_to quotes_path
+    vote.save
+    redirect_to quotes_path, notice: "Voted!" # error thrown if save fails
```

Comment:
Ah yeah I was trying to be too clever for my own good.  It will throw the error if the value is outside of the accepted but if someone has a couple tabs open and votes in one and moves to the other and votes again it will keep the first score and not alert them that their new vote didn't count.  Will change this UX around.